app/modules/individuals/models.py

```diff
@@ -116,14 +116,26 @@ class Individual(db.Model, FeatherModel):
 
     featured_asset_guid = db.Column(db.GUID, default=None, nullable=True)
 
-    version = db.Column(db.BigInteger, default=None, nullable=True)
-
     encounters = db.relationship(
         'Encounter', back_populates='individual', order_by='Encounter.guid'
     )
 
     names = db.relationship('Name', back_populates='individual', order_by='Name.created')
 
+    # TODO check that these lengths are sensible
+    comments = db.Column(db.String(length=255), nullable=True)
+    sex = db.Column(db.String(length=40), nullable=True)
+
+    # TODO what types should these be
```

Comment:
lets make them datetime.  we can fix stupid ints when we **transfer** from edm.

Reply:
Added TODO for this